src/components/PageWrapper/PageWrapper.js

```diff
@@ -13,25 +14,33 @@ class PageWrapper extends Component {
   constructor(props) {
     super(props);
 
-    this.state = { isOpen: false };
+    this.state = { navIsOpen: false };
 
     this.handleWindowClick = this.handleWindowClick.bind(this);
   }
 
+  componentDidMount() {
+    // if the nav is open and a route changes, the whole page is re-rendered
+    // before we have a change to un-block the page scroll.
+    // So do that on mount
+    blockScroll(false);
+  }
+
   handleWindowClick(e) {
     // if the user clicks, and the click is outside the nav, close it
-    if (!this.sideBarEl.contains(e.target)) {
+    if (this.sideBarEl && !this.sideBarEl.contains(e.target)) {
       this.toggleNav(false);
     }
   }
 
-  toggleNav(isOpen) {
-    this.setState({ isOpen });
+  toggleNav(navIsOpen) {
```

Comment:
Not sure about the name of this method - I'd expect toggling to reverse the current state rather than just a boolean mutator?